app/models/voxelytics/VoxelyticsDAO.scala

```diff
@@ -79,7 +79,16 @@ class VoxelyticsDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContex
           WHERE hash = $workflowHash AND _organization = $organizationId
           """.as[(String, String)])
       (name, hash) <- r.headOption
-    } yield WorkflowEntry(name, hash)
+    } yield WorkflowEntry(name, hash, organizationId)
+
+  def findWorkflowOrganizationsByHash(workflowHash: String): Fox[List[WorkflowEntry]] =
```

Comment:
As discussed in person, I’d rename this function to findWorkflowByHash (and return just the head option), and rename the function above to findWorkflowByHashAndOrganization